packages/e2e-tests/transfer-tokens/transfer-token.spec.js

```diff
@@ -0,0 +1,96 @@
+const { test, expect } = require("@playwright/test");
+const { parseNearAmount } = require("near-api-js/lib/utils/format");
+const BN = require("bn.js");
+
+const { HomePage } = require("../register/models/Home");
+const { createRandomBankSubAccount } = require("../utils/account");
+const { SendMoneyPage } = require("./models/SendMoney");
+
+const { describe, beforeAll, afterAll } = test;
+
+describe("Transferring NEAR tokens between two accounts", () => {
+    let firstAccount, secondAccount;
+
+    beforeAll(async () => {
+        [{ value: firstAccount }, { value: secondAccount }] =
+            await Promise.allSettled([
+                createRandomBankSubAccount(),
+                createRandomBankSubAccount(),
+            ]);
+    });
+
+    afterAll(async () => {
+        await Promise.allSettled([
+            firstAccount && firstAccount.delete(),
+            secondAccount && secondAccount.delete(),
+        ]);
+    });
+
+    test("navigates to send money page", async ({ page }) => {
+        test.fail(!firstAccount, "first account not successfully created");
+        const firstAccountHomePage = new HomePage(page);
+
+        await firstAccountHomePage.navigate();
+
+        await firstAccountHomePage.loginWithSeedPhraseLocalStorage(
+            firstAccount.account.accountId,
+            firstAccount.seedPhrase
+        );
+
+        await firstAccountHomePage.navigate();
+
+        await firstAccountHomePage.clickSendButton();
+
+        await expect(firstAccountHomePage.page).toMatchURL(/send-money$/);
```

Comment:
Not a blocker for this PR, but I think that we should make it a convention that any magic strings like this one should be static properties of the page model, so that both the suite and any internal navigation methods etc. can reference the same source of truth, similar to how the different `recoveryOptions` are set up on the page model.